administration/src/mui-modules/activation/ActivationPage.tsx

```diff
@@ -0,0 +1,32 @@
+import { Card } from '@mui/material'
+import { styled } from '@mui/system'
+import React, { ReactElement, useContext } from 'react'
+
+import StandaloneCenter from '../../bp-modules/StandaloneCenter'
```

Comment:
BP modules are used for internal druckerei and are not mobile friendly. Mui modules are used for application process and are mobile friendly. I just use a template component here but could also create an additional one to avoid mixing. At some point we might refactor this to only have one component library but that's gonna be kind a huge effort.

Reply:
to avoid mixing i just created a separate template component instead of using inheritance